robocop/config.py

```diff
@@ -233,32 +233,32 @@ def _create_parser(self):
                               metavar='CONFIGURABLE', help=self.HELP_MSGS['help_configure'])
         optional.add_argument('-l', '--list', action=SetListOption, nargs='?', const='', default=self.list,
                               metavar='PATTERN', help=self.HELP_MSGS['help_list'])
-        optional.add_argument('--list-configurables', action=SetListOption, nargs='?', const='',
+        optional.add_argument('-lc', '--list-configurables', action=SetListOption, nargs='?', const='',
                               default=self.list_configurables, metavar='PATTERN',
                               help=self.HELP_MSGS['help_list_confs'])
-        optional.add_argument('--list-reports', action='store_true', default=self.list_reports,
+        optional.add_argument('-lr', '--list-reports', action='store_true', default=self.list_reports,
                               help=self.HELP_MSGS['help_list_reports'])
         optional.add_argument('-o', '--output', type=argparse.FileType('w'), default=self.output,
                               metavar='PATH', help=self.HELP_MSGS['help_output'])
-        optional.add_argument('--filetypes', action=ParseFileTypes, default=self.filetypes,
+        optional.add_argument('-ft', '--filetypes', action=ParseFileTypes, default=self.filetypes,
                               help=self.HELP_MSGS['help_filetypes'])
         optional.add_argument('-t', '--threshold', action=SetRuleThreshold, default=self.threshold,
                               help=self.HELP_MSGS['help_threshold'])
         optional.add_argument('-A', '--argumentfile', metavar='PATH', help=self.HELP_MSGS['help_argfile'])
-        optional.add_argument('--ignore', action=ParseDelimitedArgAction, default=self.ignore,
+        optional.add_argument('-g', '--ignore', action=ParseDelimitedArgAction, default=self.ignore,
                               metavar='PATH', help=self.HELP_MSGS['help_ignore'])
         optional.add_argument('-h', '--help', action='help', help=self.HELP_MSGS['help_info'])
         optional.add_argument('-v', '--version', action='version', version=__version__,
                               help=self.HELP_MSGS['help_version'])
-        optional.add_argument('--verbose', action='store_true', help=self.HELP_MSGS['help_verbose'])
+        optional.add_argument('-vv', '--verbose', action='store_true', help=self.HELP_MSGS['help_verbose'])
```

Comment:
Usually -vv, or -vvv .. means "verbose but more" (for example while single --verbose print only few extra message, -vv prints every issue even if ignored etc). So it could be confusing for some people. Although we could stick to -vv if you want it to be consistent (short and long options) with other options

Reply:
Yeah, I had the same issue with that, but I think that our tool will never produce so much additional output so this -vv option is still a verbose output and it is different than -v which returns tool version. What do you think? I don't have a better idea.

Reply:
Yes, I don't want to add more levels of verbose so I believe we can go with -vv for --verbose.